bitnami/mongodb/values.yaml

```diff
@@ -348,10 +365,13 @@ labels: {}
 ## Ignored when mongodb.architecture=standalone
 ##
 replicaCount: 2
-## @param strategyType StrategyType for MongoDB(&reg;) statefulset
-## It can be set to RollingUpdate or Recreate by default.
+## @param updateStrategy.type Deployment strategy type
+## @param updateStrategy.rollingUpdate Deployment rolling update configuration parameters
+## ref: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy
 ##
-strategyType: RollingUpdate
+updateStrategy:
+  type: RollingUpdate
+  rollingUpdate: {}
```

Comment:
Including `rollingUpdate: {}` here will cause problems if other strategy type is used. Please remove it and add it in a NOTE.

Reply:
Done!